frontend/cypress/integration/talentsearch/submit-application-workflows.spec.js

```diff
@@ -0,0 +1,178 @@
+import { aliasMutation, aliasQuery } from "../../support/graphql-test-utils";
+
+const uniqueTestId = Date.now().valueOf();
+
+describe("Submit Application Workflow Tests", () => {
+  beforeEach(() => {
+    // register queries
+    cy.intercept("POST", "/graphql", function (req) {
+      aliasQuery(req, "browsePools");
+      aliasQuery(req, "getPoolAdvertisement");
+      aliasQuery(req, "getReviewMyApplicationPageData");
+      aliasQuery(req, "getApplicationData");
+      aliasQuery(req, "MyApplications");
+
+      aliasMutation(req, "createApplication");
+      aliasMutation(req, "submitApplication");
+    });
+
+    cy.getSkills().then((allSkills) => {
+      cy.wrap([allSkills[0].id]).as("testSkillIds"); // take the first ID for testing
+    });
+    cy.getGenericJobTitles().then((allGenericJobTitles) => {
+      cy.wrap([allGenericJobTitles[0].id]).as("testGenericJobTitleIds"); // take the first ID for testing
+      cy.wrap([allGenericJobTitles[0].classification.id]).as("testClassificationIds"); // take the first ID for testing
+    });
+
+    cy.loginByRole("admin").then(() => {
+      cy.get("@testGenericJobTitleIds").then((testGenericJobTitleIds) => {
+        cy.get("@testSkillIds")
+          .then((testSkillIds) => {
+            // This user must have the entire profile completed to be able to apply to a pool
+            cy.createUser({
+              email: `cypress.user.${uniqueTestId}@example.org`,
+              sub: `cypress.sub.${uniqueTestId}`,
+              roles: ["APPLICANT"],
+              currentProvince: ProvinceOrTerritory.Ontario,
+              currentCity: "Test City",
+              telephone: "+10123456789",
+              armedForcesStatus: ArmedForcesStatus.NonCaf,
+              citizenship: CitizenshipStatus.Citizen,
+              lookingForEnglish: true,
+              isGovEmployee: false,
+              hasPriorityEntitlement: false,
+              locationPreferences: WorkRegion.Ontario,
+              wouldAcceptTemporary: false,
+              expectedGenericJobTitles: {
+                sync: testGenericJobTitleIds,
+              },
+              personalExperiences: {
+                create: [
+                  {
+                    description: "Test Experience Description",
+                    details: "A Cypress test personal experience",
+                    skills: {
+                      sync: testSkillIds.map((skillId) => {
+                        return {
+                          details: `Test Skill ${skillId}`,
+                          id: skillId,
+                        };
+                      }),
+                    },
+                    startDate: FAR_PAST_DATE,
+                    title: "Test Experience",
+                  },
+                ],
+              },
+            });
+          })
+          .its("sub")
+          .as("testUserSub");
+      });
+    });
+
+    cy.getMe()
+      .its("id")
+      .then((adminUserId) => {
```

Comment:
There's a pyramid of doom starting here.  It would be nice to resolve `adminUserId`, `testClassificationIds`, and `testPoolAdvertisementId`, and `testSkillIds` all in one shot with something like a Promise.all.  I don't know if that's possible with Cypress aliases though.